### PR TITLE
chore(css): replace "timing function" with "easing function"

### PR DESCRIPTION
**Pull request by yin1999**
### Description

replace "timing function" with "easing function".

Note: I have not replaced this term in file: [`files/en-us/mozilla/firefox/releases/55/index.md`](https://github.com/mdn/content/blob/67cbfbf7a408e7180137b286247025bc40716642/files/en-us/mozilla/firefox/releases/55/index.md?plain=1#L28) and [`files/en-us/mozilla/firefox/releases/65/index.md`](https://github.com/mdn/content/blob/67cbfbf7a408e7180137b286247025bc40716642/files/en-us/mozilla/firefox/releases/65/index.md?plain=1#L41) as they are the **Historical Release Notes**.

### Motivation

See <https://www.w3.org/TR/2017/WD-css-timing-1-20170221/>, w3c is now using "Easing function" instead of "timing function"

### Related issues and pull requests

Continue: #27315, #27811

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/55/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Firefox 55 was released on August 8, 2017. This article lists key changes that a
 ### CSS
 
 - Exposed the {{cssxref("transform-box")}} property by default ([Firefox bug 1208550](https://bugzil.la/1208550)).
-- Implemented the [frames() timing function](</en-US/docs/Web/CSS/easing-function#the_frames()_class_of_timing-functions>) ([Firefox bug 1248340](https://bugzil.la/1248340)).
+- Implemented the `frames()` timing function ([Firefox bug 1248340](https://bugzil.la/1248340)).
```
Comment (yin1999) [inline files/en-us/mozilla/firefox/releases/55/index.md:28]: just remove the broken links (as the `frames()` timing function is [no longer pursuing](https://chromestatus.com/feature/5189363944128512) and this section has been removed.

File: files/en-us/mozilla/firefox/releases/65/index.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ This article provides information about the changes in Firefox 65 that will affe
   - {{cssxref("break-inside")}} is now an alias for {{cssxref("page-break-inside")}}.
 
 - The {{cssxref("overflow-wrap")}} property's `anywhere` value has been implemented ([Firefox bug 1505786](https://bugzil.la/1505786)).
-- The new step position keywords `jump-start`, `jump-end`, `jump-none`, and `jump-both` — usable inside the [`steps()` timing function](</en-US/docs/Web/CSS/easing-function#the_steps()_class_of_timing_functions>) — have been implemented ([Firefox bug 1496619](https://bugzil.la/1496619)). This also coincides with the removal of the `frames()` timing function, which was the previous way of implementing such functionality, now deprecated.
+- The new step position keywords `jump-start`, `jump-end`, `jump-none`, and `jump-both` — usable inside the [`steps()` timing function](/en-US/docs/Web/CSS/easing-function#step_easing_function) — have been implemented ([Firefox bug 1496619](https://bugzil.la/1496619)). This also coincides with the removal of the `frames()` timing function, which was the previous way of implementing such functionality, now deprecated.
```
Comment (yin1999) [inline files/en-us/mozilla/firefox/releases/65/index.md:42]: just update the broken links

**Review by estelle — APPROVED** [review]
Thank you! 🎉

**Review by bsmth — APPROVED** [review]
Thanks @yin1999 - leaving my +1 also 👍🏻

**Comment by bsmth** [comment]
Merging now 👍🏻

**Pull request merged**